vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -1102,6 +1102,12 @@ void NativeUIManager::measure(
   }
 
   auto feRootView = xamlRootView.as<winrt::FrameworkElement>();
+  if (shadowRoot.m_isWindowedPopup) {
```

Comment:
so I wonder if instead we can isolate this change purely here, that we from shadowNode to walk up parents looking for shadowRoot, but on each step up we check if the given View is a flyout/window and stop if it is...   we could add a method to ShadowNodeBase instead of m_isWindowedPopup too.  Putting stuff in ShadowNode & UIManager base things is a little less nice since that is shared with win32 and we don't need this there

Reply:
You mean redo the search for root node that UIManager is doing inside the NativeUIManager, right?  Sure. I'll work on that.

---
In reply to: [304577890](https://github.com/microsoft/react-native-windows/pull/2781#discussion_r304577890) [](ancestors = 304577890)

Reply:
I added new commits that isolate the change to ReactUWP and retrieve the appropriate XAML child view from either the Flyout or the Popup.